stmtcacher.go

```diff
@@ -76,7 +112,7 @@ type stmtCacheProxy struct {
 	db *sql.DB
 }
 
-func NewStmtCacheProxy(db *sql.DB) DBProxyBeginner {
+func NewStmtCacherProxy(db *sql.DB) DBProxyBeginner {
```

Comment:
This is part of the exported interface and cannot be renamed backward-compatibly (unfortunately). You could *add* `NewStmtCacherProxy` and document this one as deprecated.

Reply:
Alternatively you could rename StmtCacher -> StmtCache, add a `NewStmtCache` function, and document `NewStmtCacher` as deprecated.

Reply:
Ooh I like the latter. Will give that a try.